privacy.html

```diff
@@ -0,0 +1,174 @@
+            the time and date of your use of the Service, and other
+            statistics.
+        </p>
+        <br>
+        <p><strong>Cookies</strong></p>
+        <p>Cookies are files with a small amount of data that are
+            commonly used as anonymous unique identifiers. These are sent
+            to your browser from the websites that you visit and are
+            stored on your device's internal memory.
+        </p>
+        <p>This Service does not use these “cookies” explicitly. However,
+            the app may use third party code and libraries that use
+            “cookies” to collect information and improve their services.. but what about easter eggs?
+            You have the option to either accept or refuse these cookies
+            and know when a cookie is being sent to your device. If you
+            choose to refuse our cookies, you may not be able to use some
+            portions of this Service. 
+        </p>
+        <br>
+        <p><strong>Security</strong></p>
+        <p>We value your trust in providing us your
+            Personal Information, thus we are striving to use commercially
+            acceptable means of protecting it. But remember that no method
+            of transmission over the internet, or method of electronic
+            storage is 100% secure and reliable, and we cannot
+            guarantee its absolute security.
+        </p>
+        <br>
+        <p><strong>Childrens Privacy</strong></p>
+        <p>These Services do not address anyone under the age of 13.
+            We do not knowingly collect personally
+            identifiable information from children under 13 years of age. In the case
+            we discover that a child under 13 has provided
+            us with personal information, we immediately
+            delete this from our servers. If you are a parent or guardian
+            and you are aware that your child has provided us with
+            personal information, please contact us so that
+            we will be able to do necessary actions.
+        </p>
+        <br>
+        <p><strong>Changes to Privacy Policy</strong></p>
+        <p>We may update our Privacy Policy from
+            time to time. Thus, you are advised to review this page
+            periodically for any changes. We will
+            notify you of any changes by posting the new Privacy Policy on
+            this page. This policy is effective as of 2021-03-07.
+        </p>
+        <br>
+        <p><strong>Contact us</strong></p>
+        <p>If you have any questions or suggestions about our
+            Privacy Policy, do not hesitate to contact us at patrick@howpatrickareyou.com.
+        </p>
+    </div>
+    <br>
+    <div>
+        <a href="index.html"><button class="priv-play" type="button">NOW LETS PLAY!</button></a>
+    </div>
+    <br>
+    <br>
+
+    <script src="assets/js/main.js" type="module" defer></script>
+    <script src="assets/js/egg.js" defer></script>
```

Comment:
Hi Lindsay can this line be removed, as this Easter egg is only to run on the index.html page 😁